src/callbacks_step/amr_dg1d.jl

```diff
@@ -187,13 +187,13 @@ function coarsen!(u_ode::AbstractVector, adaptor, mesh::TreeMesh{1},
                 # If an element is to be removed, sanity check if the following elements
                 # are also marked - otherwise there would be an error in the way the
                 # cells/elements are sorted
-                @assert all(to_be_removed[old_element_id:(old_element_id + 2^ndims(mesh) - 1)]) "bad cell/element order"
+                @assert all(to_be_removed[old_element_id:(old_element_id + 2 - 1)]) "bad cell/element order"
 
                 # Coarsen elements and store solution directly in new data structure
                 coarsen_elements!(u, element_id, old_u, old_element_id,
                                   adaptor, equations, dg)
                 element_id += 1
```

Comment:
```suggestion
                # Increment `element_id` on the coarsened mesh by one and `skip` = 1 in 1D
                # because 2 children elements become 1 parent element
                element_id += 1
```